diagnostic.md

```diff
@@ -58,13 +62,16 @@ Suppose you want to define a mixin named `row` stored in `./row.scss`. Write the
 code to import the mixin definition in the current module.
 
 ```scss
-// your response here
+@import "row"
```

Comment:
There needs to be a semicolon after this statement, otherwise it won't be interpreted correctly.